includes/class-sensei-course-structure.php

```diff
@@ -699,6 +700,13 @@ private function sanitize_item( array $raw_item ) {
 		if ( 'module' === $raw_item['type'] ) {
 			if ( $item['id'] ) {
 				$term = get_term( $item['id'], 'module' );
+				if ( ! $term && $raw_item['slug'] ) {
+					$new_slug = $this->get_module_slug( $raw_item['title'] );
+					$term     = $this->get_module_for_teacher_by_slug( $raw_item['slug'], [ $new_slug ] );
+					if ( $term ) {
+						$item['id'] = $term->term_id;
+					}
+				}
```

Comment:
Could you add a comment that explains why this is needed? It might save someone a few hours.